apps/passport-server/src/util/util.ts

```diff
@@ -25,12 +26,10 @@ export function normalizeEmail(email: string): string {
 }
 
 /**
- * Generate a random 6-digit random token for use as a token.
+ * Generate a random 64-bit token for use in email validation/reset.
  */
 export function randomEmailToken(): string {
-  const token = (((1 + Math.random()) * 1e6) | 0).toString().substring(1);
-  if (token.length !== 6) throw new Error("Unreachable");
-  return token;
+  return randomBytes(8).toString("base64url");
```

Comment:
Does this mean the user has to enter 8 bytes of base64?  I'm against that since sometimes users can't copy/paste and need to manually copy from one screen to another.
I think 6 digits of randomness is enough with the rate limits.  I don't understand the previous algorithm for generating it so all for making it more clearly random.  Isn't randomBytes(4)%1_000_000 sufficient?

Reply:
You're right, this is probably redundant given the rate limit - I'll remove it.